frontend/packages/dev-console/integration-tests/features/guided-tour/associate-pipeline-with-application.feature

```diff
@@ -1,219 +1,120 @@
 @guided-tour
-Feature: Import an application and associate a pipeline with it tour
-              As a user, I want to take a guided tour of import an application and associate a pipeline with it feature
+Feature: Deploying an application with a pipeline tour
+              As a user, I want to take a guided tour of deploying an application with a pipeline feature
 
         Background:
             Given user is in developer perspective
+              And user is at +Add page
```

Comment:
```suggestion
              And user is at Add page
```

Reply:
Replace `+Add` with `Add` from all the scenarios from all the files.

Reply:
Updated